.travis.yml

```diff
@@ -26,7 +26,7 @@ matrix:
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 7.0
-      env: COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=3.0.*
+      env: COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=3.0.* SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
this is not a lowest dep test. You should remove this one and change the numer on line 19 to be the number of deprecations that is happening at the moment.

This means we get a failing build when we introduce new deprecated usages. As much usages are caused by Sonata, we can ignore them.